packages/loaders/package.json

```diff
@@ -26,6 +26,6 @@
   "dependencies": {
     "@pixi/core": "^5.0.4",
     "@pixi/utils": "^5.0.4",
-    "resource-loader": "^2.2.4"
+    "resource-loader": "^3.0.0"
```

Comment:
Might want this to be `v3.0.1` which includes the transpile fixes @bigtimebuddy added recently. `^3.0.0` should actually resolve to `3.0.1` but might be safer to specify that as the minimum